frontend/app/services/SubscriptionService.scala

```diff
@@ -155,6 +156,17 @@ class SubscriptionService(val tierPlanRateIds: Map[ProductRatePlan, String],
     } yield subscriptionDetails
   }
 
+  def getUsageCountWithinTerm(memberId: MemberId, unitOfMeasure: String): Future[Int] = (for {
+    account <- getAccount(memberId)
+    subscriptions <- zuoraRestService.subscriptionsByAccount(account.id)
```

Comment:
It might, but we first need to get right the logic whereby we filter and disambiguate zuora subscription records (e.g. by productRatePlanId). The next story in the backlog is about that, so I would do it then